CoreZipCode/Interfaces/PostCodeBaseService.cs

```diff
@@ -4,18 +4,52 @@
 
 namespace CoreZipCode.Interfaces
 {
+    /// <summary>
+    /// Postcode base service abstract class.
+    /// </summary>
     public abstract class PostcodeBaseService : ApiHandler
     {
+        /// <summary>
+        /// postalcode base service protected constructor.
+        /// </summary>
+        /// <param name="request">HttpClient request</param>
         protected PostcodeBaseService(HttpClient request) : base(request) { }
 
+        /// <summary>
+        /// Execute the address query by zip code.
+        /// </summary>
+        /// <param name="postcode">Postcode</param>
```

Comment:
You can improve this one to: `/// <param name="postcode">Postcode to find for</param>`. Apply that to all class parameters.